shared_governance_base/security/shared_governance_security.xml

```diff
@@ -18,10 +18,7 @@
     <record id="group_shared_governance_coordinator" model="res.groups">
         <field name="name">Coordinator</field>
         <field name="implied_ids" eval="[(4, ref('group_shared_governance_user'))]" />
-        <field
-            name="users"
-            eval="[(4, (4, ref('base.user_admin'))]"
-        />
+        <field name="users" eval="[(4, ref('base.user_admin')]" />
```

Comment:
thanks for the change, but there is now a parenthesis missing. ☺

Reply:
Well this is embarrassing.

Reply:
a good example for ci. 😁